lib/internal/async_hooks.js

```diff
@@ -226,9 +224,7 @@ function emitHook(symbol, asyncId) {
   // Use a single try/catch for all hook to avoid setting up one per
   // iteration.
   try {
-    // Using var here instead of let because "for (var ...)" is faster than let.
-    // Refs: https://github.com/nodejs/node/pull/30380#issuecomment-552948364
-    for (var i = 0; i < active_hooks.array.length; i++) {
+    for (let i = 0; i < active_hooks.array.length; i++) {
```

Comment:
I would restrain from making this change unless we have ample evidence that it's as fast as.